packages/shared/src/utils/cfd/cfd.ts

```diff
@@ -91,15 +126,20 @@ export const getAccountTypeFields = ({ category, type }) => {
     return map_mode[category][type];
 };
 
+type TGetCFDAccountDisplay = TGetCFDAccountKey & {
+    is_eu: boolean;
+    is_mt5_trade_modal?: boolean;
+};
+
 export const getCFDAccountDisplay = ({
     market_type,
     sub_account_type,
     platform,
     is_eu,
     shortcode,
     is_mt5_trade_modal,
-}) => {
-    let cfd_account_key = getCFDAccountKey({ market_type, sub_account_type, platform, shortcode });
+}: TGetCFDAccountDisplay) => {
+    let cfd_account_key: string | undefined = getCFDAccountKey({ market_type, sub_account_type, platform, shortcode });
```

Comment:
I know, we should change any logic, but isn’t it weird to return explicitly 'undefined' for this function getCFDAccountKey?
maybe we netter should have some default string return for it, maybe even empty string. then we will have no need to use 'undefined' in types.
but maybe we can skip it for now